website/src/utils/Utils.jsx

```diff
@@ -27,4 +29,11 @@ export const getStatusClass = (status) => {
 // FORMATDATE
 export const formatDate = (date) => {
     return moment(date).format('MM/DD/YYYY HH:mm')
-}
\ No newline at end of file
+}
+
+//FORMATTING BYTES TO GB
+
+export const formatByteForGB = (byte) => {
```

Comment:
Comments are easier to read if they're written next to the thing they talk about and not in all-caps.

```suggestion
// Formatting bytes to GB
export const formatByteForGB = (byte) => {
```